File: sea-shell-service-spring-web-flux/src/main/java/org/jesperancinha/shell/webflux/data/SeaShellCostumeDto.java
package org.jesperancinha.shell.webflux.data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@EqualsAndHashCode
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class SeaShellCostumeDto {
    private SeaShellTopDto topDto;
    private SeaShellLowerDto lowerDto;
}

File: sea-shell-service-spring-web-flux/src/main/kotlin/org/jesperancinha/shell/webflux/data/SeaShellCostumeDto.kt
package org.jesperancinha.shell.webflux.data

import lombok.*

@Builder
@Getter
@Setter
@EqualsAndHashCode
@ToString
@NoArgsConstructor
@AllArgsConstructor
class SeaShellCostumeDto {
    private val topDto: SeaShellTopDto? = null
    private val lowerDto: SeaShellLowerDto? = null
    private val topId: Long? = null
    private val lowerId: Long? = null
}